### PR TITLE
Update network-requirements.md

### PR DESCRIPTION
**Pull request by aduarteoutsystems**
Added a piece of explicit information that 12000+2 ports should NOT be open to the outside world.
 Separated the relevant ports to access the applications and added the explicit information of the source coming traffic

### CODE REVIEW

**Comment by aduarteoutsystems** [comment]
I read, understood and accept the OutSystems CLA

File: src/setup-maintain/setup/network-requirements.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ The OutSystems services use the following ports:
 
 * 12002 - OutSystems Scheduler Service
 
+> **Security Tips**: Those ports shouldn't be open to the internet.
```
Comment (fmalheiro) [inline src/setup-maintain/setup/network-requirements.md:29]: Consider removing the "Open Ports" section, as that information is already further down in subsequent tables.

The "Security Tip" is indeed helpful, but could be also "nearer" to the same tables - as it applies to all the ports mentioned there.

**Review by fmalheiro — CHANGES_REQUESTED** [review]

**Review by fmalheiro — APPROVED** [review]

**Review by aduarteoutsystems — COMMENTED** [review]
changed struture

**Comment by OS-johnkrug** [comment]
Hi @JoaoPBatista . I think this one is good to go. Already approved by 2 OS people.

**Pull request merged**